Sources/SourceKitLSP/Swift/CodeCompletionSession.swift

```diff
@@ -191,16 +191,15 @@ class CodeCompletionSession {
       throw ResponseError(code: .invalidRequest, message: "open must use the original snapshot")
     }
 
-    let req = SKDRequestDictionary(sourcekitd: sourcekitd)
-    req[keys.request] = sourcekitd.requests.codecomplete_open
-    req[keys.offset] = utf8StartOffset
-    req[keys.name] = uri.pseudoPath
-    req[keys.sourcefile] = uri.pseudoPath
-    req[keys.sourcetext] = snapshot.text
-    req[keys.codecomplete_options] = optionsDictionary(filterText: filterText, options: options)
-    if let compileCommand = compileCommand {
-      req[keys.compilerargs] = compileCommand.compilerArgs
-    }
+    let req = [
```

Comment:
Not a huge fan of `skd`. What if SKArray/Dictionary both had an initializer that took a sequence/dictionary?

Reply:
Changed it to `sourcekitd.dictionary(<#dict literal#>)`